webapp/src/components/preview-movies/style/row/TopRow.tsx

```diff
@@ -25,27 +25,31 @@ function TopRow({ movieList, isDataLoading }: Props) {
   }
 
   return (
-    <div className={'row'} >
+    <div className={'row'}>
       {
         isDataLoading
-          ? <RowLoading/>
+          ? (
+            <RowLoading/>
+          )
 
-          : <div className={'row_posters'} id={'top_posters'}>
-            {movieList.map((movie) => (
-              <div key={movie.title}>
-                <div className={'top_card'} style={{ width: `${window.innerWidth}px` }}>
-                  <div>
-                    <h2>{movie.title}</h2>
-                    <p>{movie.overview}</p>
-                    <Player />
+          : (
+            <div className={'row_posters'} id={'top_posters'}>
```

Comment:
Il faut éviter les ids, le but de faire des composants c'est de les réutiliser.
Cepandant les ids doivent être utilisés pour des composants uniques.

Si tu veux donner la possibilité d'ajouter un id à ton composant, il faut que ce soit une props.
En faisant ainsi, tu laisses la personne qui utilise ton composant lui ajouter un id s'il utilise ton composant qu'une fois dans sa page.

On peut en discuter si tu veux plus d'explications.

Il faut utiliser la convention BEM donc `row_posters`=> `row__posters`

```suggestion
            <div className='row__posters top__posters'>
```